### PR TITLE
大幅更改导航栏

### PR DESCRIPTION
**Pull request by zhouyiqing0304**
- [x] 我已认真阅读贡献指南 (contributing guidelines) 和社区公约 (code of conduct)，并遵循了如何参与页及格式手册页的相应规范。
有bug告诉我，别无脑删，谢谢

### CODE REVIEW

**Comment by zhouyiqing0304** [comment]
image
image
image

**Comment by boomzero** [comment]
Please make the popover stuff a separate PR, thanksOn Aug 1, 2024, at 18:59, zhouyiqing0304 ***@***.***> wrote:﻿
image.png (view on web)
image.png (view on web)
image.png (view on web)

**Comment by boomzero** [comment]
Okay, looks greatOn Aug 1, 2024, at 18:59, zhouyiqing0304 ***@***.***> wrote:﻿
image.png (view on web)
image.png (view on web)
image.png (view on web)

**Comment by zhouyiqing0304** [comment]
@boomzero https://github.com/XMOJ-Script-dev/XMOJ-Script/pull/588
pr过了

File: XMOJ.user.js [hunk 1]
```diff
@@ -1044,7 +1136,8 @@ async function main() {
                         "ID": "ImproveACRate", "Type": "A", "Name": "自动提交已AC题目以提高AC率"
                     }, {"ID": "AutoO2", "Type": "F", "Name": "代码提交界面自动选择O2优化"}, {
                         "ID": "Beautify", "Type": "F", "Name": "美化界面", "Children": [{
-                            "ID": "NewBootstrap", "Type": "F", "Name": "使用新版的Bootstrap样式库*"
+                            "ID": "NewTopBar", "Type": "F", "Name": "使用重写的顶栏"
```
Comment (PythonSmall-Q) [inline XMOJ.user.js:1139]: 改为：使用“新版”导航栏

可以括号里标注干了啥

**Review by PythonSmall-Q — CHANGES_REQUESTED** [review]

File: XMOJ.user.js (hunk 1 above)
Comment (zhouyiqing0304) [reply XMOJ.user.js:1139]: ？
什么意思

Comment (zhouyiqing0304) [reply XMOJ.user.js:1139]: 哦

Comment (PythonSmall-Q) [reply XMOJ.user.js:1139]: 描述不准确

Comment (PythonSmall-Q) [reply XMOJ.user.js:1139]: 写清楚干啥了

Comment (PythonSmall-Q) [reply XMOJ.user.js:1139]: 双引号去掉

**Comment by boomzero** [comment]
Does https://getbootstrap.com/docs/5.3/components/navbar/#placement do this?

**Comment by PythonSmall-Q** [comment]
@boomzero 链接404

**Comment by zhouyiqing0304** [comment]
@boomzero @PythonSmall-Q 
可以了

**Comment by zhouyiqing0304** [comment]
> 可以用一行代码实现

说说看？

**Comment by zhouyiqing0304** [comment]
@boomzero 最好写出来我学习一下）

**Comment by PythonSmall-Q** [comment]
https://github.com/XMOJ-Script-dev/XMOJ-Script/pull/592

**Comment by PythonSmall-Q** [comment]
圆角，模糊有吗？

**Comment by boomzero** [comment]
I will not merge custom CSS, thank you.

**Comment by boomzero** [comment]
And can be added with more code

**Pull request merged**